solr/core/src/java/org/apache/solr/client/solrj/embedded/EmbeddedSolrServer.java

```diff
@@ -164,7 +164,8 @@ public NamedList<Object> request(SolrRequest<?> request, String coreName)
     if (handler != null) {
       try {
         SolrQueryRequest req =
-            _parser.buildRequestFrom(null, request.getParams(), getContentStreams(request));
+            _parser.buildRequestFrom(
```

Comment:
I hate that we have `_parser`, I mean, we don't use that as a global pattern to define private variables!  Renaming the various `_myObject` would be for another ticket, I know ;-).

Reply:
okay, looked more, and yeah, i guess one method is used from http, but the other isn't cause it's for tests etc....  okay.

Reply:
i wish there was a better way than passing `null` around everywhere.

Reply:
That underscore is from 2008 by Ryan McKinley -- long time ago :-)
BTW I really appreciate that you are caring to these minor matters,

Reply:
Goign to bring it up at community meeting tomorrow...